packages/driver/src/cross-origin/cypress.ts

```diff
@@ -135,6 +136,19 @@ const setup = (cypressConfig: Cypress.Config, env: Cypress.ObjectLike) => {
   // @ts-ignore
   Cypress.isCy = cy.isCy
 
+  // this is "valid" inside the cy.origin() callback (as long as the experimental
+  // flag is enabled), but it should be replaced by the preprocessor at runtime
+  // with an actual require() before it's run in the browser. if it's not,
+  // something unexpected has gone wrong
+  Cypress.require = () => {
+    // @ts-ignore
+    if (!Cypress.config('experimentalOriginDependencies')) {
+      $errUtils.throwErrByPath('require.invalid_without_flag')
+    }
+
+    $errUtils.throwErrByPath('require.invalid_inside_origin')
```

Comment:
This line shouldn't be reachable right?

Reply:
Right. Is there some I can add to the comment above this to make that clearer?

Reply:
I don't know if we have to, i was just double checking.